### DISCUSSION TITLE
How to suppress SQL error in Production environment?

### DISCUSSION THREAD

**Discussion by joelamltc** [directus/directus#16312] [Archive]
In production, due to security reasons, we want to suppress SQL-related errors and prevent the errors from exposing to the public while still logging them to the server log.

Is there a way to do it in Directus? 

<img width="1222" alt="image" src="https://user-images.githubusercontent.com/13143636/199903448-69d9597c-38eb-4a7e-9695-613e63cdd8c0.png">

Thanks!

**Comment by rijkvanzanten** [comment]
@joelamltc Mind opening a new Issue detailing the setup? This doesn't sound like expected / intended behavior to me
